SMCalloutView.m

```diff
@@ -25,9 +25,9 @@ @interface UIView (SMFrameAdditions)
 #define SUBTITLE_TOP 28 // the top of the subtitle, when present
 #define SUBTITLE_HEIGHT 15 // subtitle height, fixed
 #define BETWEEN_ACCESSORIES_MARGIN 7 // margin between accessories when no title/subtitle is present
-#define CONTENT_VIEW_MARGIN 12 // margin around content view when present
+#define CONTENT_VIEW_MARGIN 12 // margin around content view when present, can be overridden with contentViewMargin
```

Comment:
Could this be removed entirely, and instead have `contentViewInset` initialized to a value of `12,12,12,12`?